src/app/api/api_v1/endpoints/detections.py

```diff
@@ -165,19 +165,24 @@ async def fetch_unlabeled_detections(
     def get_url(detection: Detection) -> str:
         return bucket.get_public_url(detection.bucket_key)
 
+    async def get_url_with_bucket(detection: Detection) -> str:
+        camera = cast(Camera, await cameras.get(detection.camera_id, strict=True))
```

Comment:
Considering the size of the potential result, this could become quite slow. We've had to solve this on the old-production branch. I suggest taking the same approach : https://github.com/pyronear/pyro-api/blob/old-production/src/app/api/endpoints/events.py#L149

(otherwise if there are 150 detections, we'll have to do 150 invidual camera fetch)

Reply:
ok I have implemented a pagination parameter in the endpoint (and the fetch_all function) is it ok for you ?

Reply:
On paper yes, but what is implemented on old-production was proofed with platform interactions and extremely fast. Pagination will for sure needed at some point, but for now, for the sake of "let's do what we know works well", I'd suggest taking only adapting that code. If you feel like we should introduce pagination now:
- add a param to change the num of events (https://github.com/pyronear/pyro-api/blob/old-production/src/app/api/endpoints/events.py#L165), but enforce that it cannot be bigger than 15 for instance
- add another param to change the num of consecutive alerts per event for the GIF (https://github.com/pyronear/pyro-api/blob/old-production/src/app/api/endpoints/events.py#L221), but enforce that it cannot be bigger than 10 for instance

Considering this is only for the platform and for now, it fills that need, I think those param and pagination aren't necessary but I'll let you be the judge of that :ok_hand: